scripts/performance/memory-performance/compute-memory-performance.js

```diff
@@ -0,0 +1,115 @@
+const puppeteer = require('puppeteer')
+const { timeout } = require('../../lib/execution-utils')
+const { fetchPR, LOCAL_BRANCH } = require('../../lib/git-utils')
+const NUMBER_OF_RUNS = 40 // Rule of thumb: 30 runs should be enough to get a good average
```

Comment:
**Nit**: the comment mentions 30 runs, but the constant is 40. To make sure we don't have incorherence between the doc and the code, you could remove the number in the comment:

```suggestion
const NUMBER_OF_RUNS = 40 // Rule of thumb: this should be enough to get a good average
```